fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -226,7 +228,6 @@ public class LuceneIndexTest extends FDBRecordStoreTestBase {
     private static Index complexPartitionedIndex(final Map<String, String> options) {
         return new Index("Complex$partitioned",
                 concat(function(LuceneFunctionNames.LUCENE_TEXT, field("text")),
-                        function(LuceneFunctionNames.LUCENE_STORED, field("timestamp")),
```

Comment:
This is a nice win that I did not foresee with the removal of the `anchorTimestamp`